server/controllers/monitoringConditions/checkAnswersController.test.ts

```diff
@@ -722,128 +658,77 @@ describe('MonitoringConditionsCheckAnswersController', () => {
-                  visuallyHiddenText: 'address line 1',
-                },
-              ],
-            },
-          },
-          {
-            key: {
-              text: 'Address line 2',
-            },
-            value: {
-              text: 'Line 2',
-            },
-            actions: {
-              items: [
-                {
-                  href: paths.MONITORING_CONDITIONS.INSTALLATION_ADDRESS.replace(':orderId', order.id),
-                  text: 'Change',
-                  visuallyHiddenText: 'address line 2',
-                },
-              ],
-            },
-          },
-          {
-            key: {
-              text: 'Address line 3',
-            },
-            value: {
-              text: '',
-            },
-            actions: {
-              items: [
-                {
-                  href: paths.MONITORING_CONDITIONS.INSTALLATION_ADDRESS.replace(':orderId', order.id),
-                  text: 'Change',
-                  visuallyHiddenText: 'address line 3',
-                },
-              ],
-            },
-          },
-          {
-            key: {
-              text: 'Address line 4',
+              text: 'Where will the installation of the electronic monitoring device take place?',
             },
             value: {
-              text: '',
+              html: 'Line 1, Line 2, Postcode',
             },
             actions: {
               items: [
                 {
                   href: paths.MONITORING_CONDITIONS.INSTALLATION_ADDRESS.replace(':orderId', order.id),
                   text: 'Change',
-                  visuallyHiddenText: 'address line 4',
+                  visuallyHiddenText: 'where will the installation of the electronic monitoring device take place?',
                 },
               ],
             },
           },
+        ],
+        curfewReleaseDate: [
           {
             key: {
-              text: 'Postcode',
+              text: 'What date is the device wearer released from custody?',
             },
             value: {
-              text: 'Postcode',
+              text: '11/11/2024',
             },
             actions: {
               items: [
                 {
-                  href: paths.MONITORING_CONDITIONS.INSTALLATION_ADDRESS.replace(':orderId', order.id),
+                  href: paths.MONITORING_CONDITIONS.CURFEW_RELEASE_DATE.replace(':orderId', order.id),
                   text: 'Change',
-                  visuallyHiddenText: 'postcode',
+                  visuallyHiddenText: 'what date is the device wearer released from custody?',
                 },
               ],
             },
           },
-        ],
-        curfew: [
           {
             key: {
-              text: 'Release date',
+              text: 'On the day of release, what time does the curfew start?',
             },
             value: {
-              text: '11/11/2024',
+              text: 'Invalid Date',
             },
```

Comment:
🙄

Reply:
Fixing this [here](https://github.com/ministryofjustice/hmpps-electronic-monitoring-create-an-order/pull/334)